server.js

```diff
@@ -16,12 +21,172 @@ const app = express();
 app.use(cors());
 app.use(express.json());
 
+
+const ThoughtsSchema = new mongoose.Schema({
+	//in this object specify every single thing about this property(name)
+	name: {
```

Comment:
You are not accepting `name` as a property for posting to the db (only message). Perhaps your should remove the name property from the schema if you are not using it.